### DISCUSSION TITLE
Query params (filtering) on tile endpoints - testing / feedback

### DISCUSSION THREAD

**Discussion by ARolek** [go-spatial/tegola#895] [General]
This discussion is to gather feedback on the [recently merged PR](https://github.com/go-spatial/tegola/pull/867) to support filtering vector tiles based on query string params. Since this is a rather large feature we want to allow for testing prior to cutting an official release. If you're interested in testing this feature the [build artifacts can be grabbed from the CI](https://github.com/go-spatial/tegola/actions/runs/3595608605). Configuration instructions can be found in the PR [here](https://github.com/go-spatial/tegola/pull/867#issue-1279543129). 

This feature was authored by @bemyak.

**Comment by Jonny-Johnson** [comment]
Was this feature ever properly merged and released? From the README, it looks like it. But the PRs seem to have been closed without being merged?

**Reply by ARolek** [reply]
@Jonny-Johnson this feature is available in tegola. There's an example in the README.md here: https://github.com/go-spatial/tegola?tab=readme-ov-file#example-config-using-postgres-12--postgis-30-st_asmvt
